src/components/SpecificationBlock.jsx

```diff
@@ -1,14 +1,14 @@
 export default function SpecificationBlock({ heading, children }) {
   return (
-    <div className="cmp-specification-block__wrapper">
+    <div className="cmp-specifications-block__wrapper">
```

Comment:
Is there a reason you chose to switch the class name to plural when the component name is not?

Reply:
In our `[id].pages.jsx` file, the `div` that contains all of these `SpecificationBlock`s was named `cmp-specifications-block`. To get my BEM/SCSS nesting to work properly, I had to change this name to be plural to match. Should I do it the opposite way?